gui/autotests/.gitignore

```diff
@@ -0,0 +1,72 @@
+# This file is used to ignore files which are generated
```

Comment:
Any reason not to put this into the top-level gitignore?

Reply:
This is a generated file by the QT Creator and is test specific